lib/govuk_connect/cli.rb

```diff
@@ -906,6 +948,25 @@ def types
         )
       end,
 
+      "amazonmq" => proc do |target, environment, args, extra_args, _options|
+        check_for_additional_arguments("amazonmq", args)
+        check_for_additional_arguments("amazonmq", extra_args)
+
+        target ||= "publishingmq.#{environment}.govuk-internal.digital"
+
+        info "You'll need to login as the RabbitMQ #{bold('root')} user."
+        info "Get the password from govuk-aws-data, for example:\n\n"
+        info "  # from the root directory of govuk-aws-data repository\n\n"
+        info "  #{bold(amazonmq_root_password_command(environment))}"
+        print_empty_line
+
+        remote_port_forward(
```

Comment:
Forgive my ignorance - how is this different from the [existing RabbitMQ call to `ssh`](https://github.com/alphagov/govuk-connect/blob/7b506a0a8c1ed02a8b3dbe42ff560ad9559e16c9/lib/govuk_connect/cli.rb#L939-L949)? From a quick look they appear to be doing very similar things, did you try reusing the [existing `ssh` method](https://github.com/alphagov/govuk-connect/blob/7b506a0a8c1ed02a8b3dbe42ff560ad9559e16c9/lib/govuk_connect/cli.rb#L530-L577)?

Reply:
Ah ok, makes sense - thanks for the explanation!